doc/toolbox.1.md

```diff
@@ -87,40 +87,18 @@ by option **log-level**.
 Same as `--log-level=debug`. Use `-vv` to include `--log-podman`.
 
 ## COMMANDS
-
-Commands for working with toolbox containers and images:
-
-**toolbox-create(1)**
-
-Create a new toolbox container.
-
-**toolbox-enter(1)**
-
-Enter a toolbox container for interactive use.
-
-**toolbox-help(1)**
-
-Display help information about Toolbox.
-
-**toolbox-init-container(1)**
-
-Initialize a running container.
-
-**toolbox-list(1)**
-
-List existing toolbox containers and images.
-
-**toolbox-rm(1)**
-
-Remove one or more toolbox containers.
-
-**toolbox-rmi(1)**
-
-Remove one or more toolbox images.
-
-**toolbox-run(1)**
-
-Run a command in an existing toolbox container.
+The following commands correspond to markdown files which will help you with execution.
+
+| Command                                          | Description                                                                 |
+| ------------------------------------------------ | --------------------------------------------------------------------------- |
+| [toolbox-create(1)](toolbox-create.1.md)         | Create a new toolbox container                                              |
```

Comment:
These Markdown files are primarily used to generate troff formatted manuals with `go-md2man(1)`, and these changes generate broken files.

It will be good if we can link to these Markdown files from the Toolbx [website](https://containertoolbx.org/), but that will require figuring out how to add links without breaking the generated manuals.

Reply:
Here's a screenshot showing the breakage:

![image](https://github.com/containers/toolbox/assets/3208107/0bcf04fc-aee4-4da3-8291-4e7714162c02)